serverless/scripts/remove.js

```diff
@@ -29,9 +29,35 @@ async function remove() {
 
   cli.action.start('Removing Sync Services');
   const syncServices = await client.sync.services.list();
-  syncServices.forEach(function(service) {
+  syncServices.forEach(async function(service) {
     if (service.friendlyName.includes(constants.SYNC_SERVICE_NAME_PREFIX)) {
-      client.sync.services(service.sid).remove();
+      cli.action.start(`Removing Sync Service: ${service.friendlyName}`);
+      const syncService = client.sync.services(service.sid);
```

Comment:
Should this be called `syncClient`? Note that the `forEach` parameter is already a "service". I think this is the convention @timmydoza started.

Reply:
I think `syncService` is the correct descriptor here. The `syncService` is a `SyncContext` object according to the API docs so I think the operations match the descriptor. However, I can be more explicit and call it a `liveSyncService` which is more accurate. thoughts?

Reply:
I don't think we need to prepend with `live`. 

What doc are you referring to?

If I look at the helper library source I see `ServiceInstance` and `ServiceContext`. Interesting that we aren't using either of those terms.

What is confusing to me is how we use `service` to create `syncService`. Which I assume is why Tim M used the convention `syncClient`. To distinguish between an instance and context. We shouldn't call both an instance and a context a service right? 

What would we call these things here?
https://github.com/twilio/twilio-live-interactive-video/blob/4bb4e68d3fd7afdea91f1c341027312d2fd68d5c/serverless/functions/create-stream.js#L116-L123

I'm inclined to use the established convention until Tim M returns or there is an obvious better way. 

But I won't hold this up since it is just semantics and I am no JS pro.